system/runners/BDDRunner.cfc

```diff
@@ -166,6 +166,7 @@ component extends="testbox.system.runners.BaseRunner" implements="testbox.system
 			// prepare threaded names
 			var threadNames = [];
 			// threaded variables just in case some suite is async and another is not.
+			local.thread = {};
```

Comment:
This is incorrect. It is the `thread` scope

Reply:
Lucee's implicit variable debugging says it's resolving it from the variables scope? 

C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1079 | variables | THREAD | 129
-- | -- | -- | -- | --
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1080 | variables | THREAD | 129
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1081 | variables | THREAD | 129
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1084 | variables | THREAD | 129
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1087 | variables | THREAD | 129
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1091 | variables | THREAD | 246
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1091 | variables | NEXTCLOSURE | 80
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1096 | variables | THREAD | 46
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1100 | variables | NEXTCLOSURE | 43
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1102 | variables | NEXTSPECINFO | 43
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1104 | variables | THREAD | 89
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1105 | variables | THREAD | 89
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1106 | variables | THREAD | 89
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1107 | variables | _CLOSUREINDEX | 43
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1108 | variables | GENERATEAROUNDEACHCLOSURESSTACK | 89
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1109 | variables | NEXTSPECINFO | 43
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1110 | variables | NEXTSPECINFO | 43
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1111 | variables | NEXTSPECINFO | 43
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1112 | variables | NEXTSPECINFO | 43
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1114 | variables | THREAD | 89
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1115 | variables | NEXTCLOSURE | 43
C:\lucee\tomcat\webapps\ROOT\testbox\system\BaseSpec.cfc | 1344 | variables | THIS | 16

Reply:
Same as above